src/template-loader.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {assert} from '../lib/assert';
+import {getNamespace} from '../lib/namespace';
+import {successfulFetch} from './utils/xhr';
+
+export const templateFileUrl = (templateName, filename) =>
+  `/static/templates/${templateName}/${filename}`;
+
+const {s} = getNamespace('editor');
```

Comment:
Instead of setting an incorrect namespace here, pass the script element to the `TemplateLoader` as selected from `Editor`.